.github/workflows/publish.yml

```diff
@@ -35,22 +35,22 @@ jobs:
       run: |
         pytest
 
-    - name: Build package
+    - name: Test install package
       env:
         PYTHON_VERSION: ${{ matrix.python-version }}
       run: |
         python setup.py bdist_wheel --python-tag py${PYTHON_VERSION//.}
 
     - name: Test install package
       run: |
-        mkdir test_install
-        cd test_install
-
         pip install virtualenv
         virtualenv installable
         source installable/bin/activate
 
-        pip install $(find ${GITHUB_WORKSPACE}/dist -iname "*.whl")
+        pip install dist/$(ls dist)
```

Comment:
if this fails, the import below still works, which defeats its purpose :)

Reply:
Not true? We use `cd` just the same as you did before
See lines below
```
        mkdir test_install
        cd test_install
        python -c "import datastream"
```

Reply:
sorry stopped interpreting what was happened after the pip install
it feels a bit more hacky environment-wise since you are half the time in a potentially tainted pwd and you have to ensure that the pwd is not important during the whl installation (I know it usually is, but not sure that this is guaranteed)
hmm, now when I say it out loud its the same issue with the previous version as well since it could for example install a file under current pwd for some reason and the only reason it works is since the file is placed under the test_install folder.
but its a bit far fetched I guess
but the main point stands, feels like less logic to be under the same pwd during this test

Reply:
but this is a meh, 50/50 don't really care either way